server/src/test/java/keywhiz/service/resources/automation/v2/SecretResourceTest.java

```diff
@@ -153,30 +166,6 @@ public class SecretResourceTest {
   //---------------------------------------------------------------------------------------
   // createOrUpdateSecret
   //---------------------------------------------------------------------------------------
-
```

Comment:
the test config file now specifies that the owner should be inferred so this test does not make sense any more